twice_decorator.py

```diff
@@ -0,0 +1,15 @@
+from functools import wraps
+
+def twice_decorator(func):
+    @wraps(func)
+    def function_wrapper(*args, **kargs):
+        func(*args, **kargs)
+        func(*args, **kargs)
+
+    return function_wrapper
+
+@twice_decorator
+def f(i):
```

Comment:
Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing

Rename 'i' and 'f'